akka-actor/src/main/resources/reference.conf

```diff
@@ -94,6 +94,11 @@ akka {
   # terminate the ActorSystem itself, with or without using CoordinatedShutdown.
   jvm-shutdown-hooks = on
 
+  # Version must be the same across all modules and if they are different the startup
+  # will fail. It's possible but not recommended, to disable this check, and only log a warning,
```

Comment:
Maybe make this more threatening

Reply:
remove it from migration guide? good to still have the possibility if there is some reason we haven't thought of (e.g. some weird ci build)